tests/test_project.py

```diff
@@ -136,13 +151,41 @@ def get_project_fixture():
         db.session.flush()
         db.session.commit()
 
+        image1 = Image(project_id=1, image_name='cracked_building.png')
+        db.session.add(image1)
+        db.session.flush()
+        db.session.commit()
+
+        label1 = Label(label_name='asdf1', project_id=project.id)
+        db.session.add(label1)
+        label2 = Label(label_name='asdf2', project_id=project.id)
+        db.session.add(label2)
+        db.session.flush()
+        db.session.commit()
+
+        annotation1 = Annotation(
+            image_id=1, project_id=1, creator_id=1, label_id=1, data=b'1234',
+            vector=b'1234')
```

Comment:
why does this vector get its own line?

Reply:
The coding style as dictated by the Google Python Style Guide (http://google.github.io/styleguide/pyguide.html) says that there must not be more than 80 chars per line. That's why the vector property is moved to a new line.